mesonbuild/coredata.py

```diff
@@ -576,6 +576,8 @@ def read_cmd_line_file(build_dir, options):
     properties = config['properties']
     if options.cross_file is None:
         options.cross_file = properties.get('cross_file', None)
+    if not options.native_file:
+        options.native_file = properties.get('native_file', [])
```

Comment:
I don't think this is going to deserialize into a list of strings. But rather to a string `'['/path1', '/path2']'`

Reply:
let me go look.

Reply:
You are correct, I'll fix that.